akka-stream/src/main/scala/akka/stream/impl/io/IOSinks.scala

```diff
@@ -19,7 +19,7 @@ import scala.concurrent.{ Future, Promise }
  * Creates simple synchronous (Java 6 compatible) Sink which writes all incoming elements to the given file
```

Comment:
I removed both comments. This wasn't Java 6 compatible from a moment when `FileChannel` was used.